packages/wonder-blocks-dropdown/src/components/__tests__/action-menu.test.js

```diff
@@ -214,11 +209,11 @@ describe("ActionMenu", () => {
                 <OptionItem label="Toggle B" value="toggle_b" />
             </ActionMenu>,
         );
-        menu.find(DropdownOpener).simulate("click");
+        userEvent.click(screen.getByRole("button"));
 
         // Act
-        const optionItem = menu.find(OptionItem).at(0);
-        optionItem.props().onToggle("toggle_a");
+        const optionItem = screen.getByText("Toggle A");
+        userEvent.click(optionItem);
```

Comment:
Why don't we have to wait for the dropdown to appear like we do in the `SingleSelect` tests?